### DISCUSSION TITLE
Lost somewhere

### DISCUSSION THREAD

**Discussion by TriftenX** [UltimaHoarder/UltimaScraper#1207] [General]
@DIGITALCRIMINAL 

Been fighting with this for a hour or so.

I downloaded the most recent commit, ran to proces, set config and auth files, VS asked me to update gecjodriver in the OFlofgin file, ran it again, noew i get this:

Auth (V1) Attempt 1/10
Access denied.
Auth (V1) Attempt 2/10
Access denied.
Auth (V1) Attempt 3/10
Access denied.
Auth (V1) Attempt 4/10
Access denied.
Auth (V1) Attempt 5/10
Access denied.
Auth (V1) Attempt 6/10
Access denied.
Auth (V1) Attempt 7/10
Access denied.
Auth (V1) Attempt 8/10
Access denied.
Auth (V1) Attempt 9/10
Access denied.
Auth (V1) Attempt 10/10
Access denied.
Opening Browser
Scraping Paid Content
Scraping Subscriptions
There's nothing to scrape.
Archive Completed in 0.13 Minutes
Pausing scraper for 0 seconds.

Any advice, did i miss something?

**Comment by LunarPenguin** [comment]
Make sure your auth settings are correct, it looks like they are not correct atm

make sure there are actual models to be scraped.

**Comment by TriftenX** [comment]
Yeah. I was actually trying to delete this thread. I tried updating, then it required, gecko. I did a fresh install and it turned out all my auth info had changed.

**Reply by LunarPenguin** [reply]
> I did a fresh install and it turned out all my auth info had changed

Yes this unfortunately happens alot, there is not alot we can do about it, but it causes alot of issues to be submitted
